libs/ui/breadcrumb/helm/src/lib/breadcrumb.directive.ts

```diff
@@ -0,0 +1,19 @@
+import { Directive, computed, input } from '@angular/core';
+import { hlm } from '@spartan-ng/ui-core';
+import { ClassValue } from 'clsx';
+
+@Directive({
+	selector: '[hlmBreadcrumb]',
+	standalone: true,
+	host: {
+		role: 'navigation',
+		'[class]': '_computedClass()',
+		'[attr.aria-label]': 'ariaLabel()',
+	},
+})
+export class HlmBreadcrumbDirective {
+	public readonly userClass = input<ClassValue>('', { alias: 'class' });
+	public readonly ariaLabel = input<string>('pagination', { alias: 'aria-label' });
```

Comment:
Change the text back to `breadcrumb` instead of `pagination`

Reply:
I fixed it. Thanks